pkg/controller/machine/controller.go

```diff
@@ -168,14 +168,18 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 				return reconcile.Result{Requeue: true, RequeueAfter: requeueErr.RequeueAfter}, nil
 			}
 		}
+		return reconcile.Result{}, err
```

Comment:
I find `return reconcile.Result{}, nil` is more clear.  Is this convention used elsewhere?

Reply:
nm, this is correct